### ISSUE TITLE
.DS_store always recognized

### ISSUE THREAD

**Issue by Jaspur** [thephpleague/flysystem#467]
Every time I manually set an if-statement to exclude .DS_store-files. Isn't this something that can be stripped by default?

**Comment by RobQuistNL** [comment]
Why would you want to strip files that are really there? If you don't want the DS_store files there you should delete them, right? Maybe you can make a general option to hide "hidden" dotfiles... But if you ask me this should not be stripped by default.

**Comment by frankdejonge** [comment]
This is the type of opinionated code that doesn't belong in flysystem itself. I'd suggest either to create a plugin for this or to resolve this in a wrapper of some sort. If this is resolved in the listContents call this would also mean it should get deleted implicitly when deleting a directory for example. This will create much unneeded complexity. This would also have to be made either configurable or as an optin/optout flag, which is IMO quite nasty.

TLDR; while it's a common case, this should not be resolved within flysystem.
